EIPS/eip-7704.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 7704
```

Comment:
```suggestion
eip: 7707
```
Assigning next sequential EIP/ERC/RIP number.  7704 had already been issued.

Please also update the filename.